backend/README.md

```diff
@@ -27,14 +27,20 @@ the regular base application.
 The project requires a couple of environment variables to run, if you want to develop on this codebase.
 Setting values for these variables can be done with a method to your own liking.
 
-| Variable          | Description                                                    |
-|-------------------|----------------------------------------------------------------|
-| DB_HOST           | Url of where the database is located                           |
-| POSTGRES_USER     | Name of the user, needed to login to the postgres database     |
-| POSTGRES_PASSWORD | Password of the user, needed to login to the postgres database |
-| POSTGRES_HOST     | IP adress of the postgres database                             |
-| POSTGRES_DB       | Name of the postgres database                                  |
-| API_HOST          | Location of the API root                                       |
+| Variable                               | Description                                                                         |
+|----------------------------------------|-------------------------------------------------------------------------------------|
+| DB_HOST                                | Url of where the database is located                                                |
+| POSTGRES_USER                          | Name of the user, needed to login to the postgres database                          |
+| POSTGRES_PASSWORD                      | Password of the user, needed to login to the postgres database                      |
+| POSTGRES_HOST                          | IP adress of the postgres database                                                  |
+| POSTGRES_DB                            | Name of the postgres database                                                       |
+| API_HOST                               | Location of the API root                                                            |
+| CLIENT_ID                              | [Client id](https://learn.microsoft.com/nl-nl/entra/identity-platform/v2-protocols) |
+| CLIENT_SECRET                          | Client's secret                                                                     |
+| CODE_VERIFIER                          | Code verifier                                                                       |
```

Comment:
You haven't explained what this is, your explanation is the name

Reply:
Cause I don't know what it is :skull:

Reply:
https://github.com/SELab-2/UGent-3/pull/357/commits/115417609c6a0b358e1ae30ee816d1097ee1334d